.travis.yml

```diff
@@ -1,185 +0,0 @@
-language: c
```

Comment:
probably, appveyor.yml can also be removed in the same vein. 😄

Reply:
LOL, turns out that the appveyor.yaml got [nuked by mistake](https://github.com/Snaipe/Criterion/commit/017b3003f8bbb452502f29ab3981855191df7581) when I merged the meson branch. Way to be proactive!

Reply:
Great! they can also be nuked from https://github.com/Snaipe/Criterion/settings/installations and https://github.com/Snaipe/Criterion/settings/hooks to hide them from CI run in PRs.